src/components/display/fewlines/Header/Header.tsx

```diff
@@ -19,6 +24,10 @@ const Flex = styled.div`
   align-items: center;
   margin: ${({ theme }) => `${theme.spaces.xs} ${theme.spaces.xs}`};
 
+  img {
+    cursor: pointer;
+  }
```

Comment:
even when the `img` is inside an `a`?

Reply:
Nice catch. I added that because at first I used a `onClick` on the `<img/>` before realizing that I needed to use a Nexj.js `Link` component.